cyberdrop_dl/utils/data_enums_classes/supported_domains.py

```diff
@@ -1,5 +1,8 @@
 
-from enum import Enum
+from aenum import Enum, extend_enum
```

Comment:
Do not use `aenum`, make a new class, `SupportedSitesDebug` that inherits from `SupportedSites` and choose which one to use base on the pre-release tags check in the cookie extraction logic

```python
class SupportedSitesDebug(SupportedSites):
   simpcity = "simpcity"
```

In the pydantic config model, the type annotation should be with `SupportedSitesDebug`, but the default value should be from the regular `SupportedSites`

Reply:
Yeah I wasn't sure about that because it will show all the sites the input can support. So I was trying to hide the simpcity for non dev versions

Reply:
I think I found a solution